XWebView/XWebView.h

```diff
@@ -54,4 +54,10 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)invokeWithTarget:(id)target;
 @end
 
+// Special init which can't be reference directly in Swift, but cannot be a protocol either.
+@interface _InitSelector: NSObject
+// Init with script
+- (id)initByScriptWithArguments:(va_list)args;
```

Comment:
No problem, bit unclear what you want me to do here though, replace with NSArray?

I've just tried that and everything seems OK!